docs/examples/lynx/checkbox/preview.css

```
@@ -0,0 +1,17 @@
+/* biome-ignore lint/correctness/noUnknownTypeSelector: Lynx의 page 요소입니다. */
+page {
```

Stylelint (17.14.0)

[error] Unknown type selector "page" (selector-type-no-unknown)

(selector-type-no-unknown)